op-chain-ops/eof-crawler/eof_crawler.go

```diff
@@ -0,0 +1,135 @@
+package main
```

Comment:
nit: this should expose functions that are imported into `cmd/eof-crawler/main.go` and the main binary should be there